wire-ios/Wire-iOS/Sources/Authentication/Event Handlers/AuthenticationEventResponderChain.swift

```diff
@@ -51,7 +52,8 @@ final class AuthenticationEventResponderChain {
     enum EventType: CustomStringConvertible {
 
         case wireAuthenticationModuleComplete((AuthenticationResult, RegistrationAnalyticsTrackingConsent))
-        case flowStart(NSError?, Int)
+        case logoutRequested(deleteData: Bool)
```

Comment:
The new reauth screens have a button to logout which is handled outside the module by way of an outbound event.